app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource
   def index
     @user = User.find(params[:user_id])
```

Comment:
### Optional
- [ ] It would be better if we fix the `n+1` query problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS). Here is a quick fix for that as seen below

```
@posts = @user.posts.includes(:comments)
instead of @posts = @user.posts
```